app/lib/actions/katello/repository/check_matching_content.rb

```diff
@@ -23,8 +23,10 @@ def run
             yum_metadata_files = yum_metadata_files_match?(source_repo, target_repo)
             checksum_match = (target_repo.saved_checksum_type == source_repo.saved_checksum_type)
 
+            published = target_repo.backend_service(SmartProxy.pulp_primary).published?
```

Comment:
It appears that this will break the pulp2 support because this new method is not added to the Katello::Pulp::Repository. Is it intended?

Reply:
i had noticed this too a while back on my development environment, but this commit went into katello 4.0 which no longer includes pulp2.  Due to that, i didn't file a bug as it shouldn't ever show up for a user.  Under what version/environment are you seeing this be an issue?

Reply:
I don't see any issue. I just notice this change when working on a patch (https://github.com/Katello/katello/pull/9348) in the master branch for Satellite 6.9 so I thought I will just ask to confirm you guys know about it.

Reply:
yes, this could NOT be cherry picked to 6.9 (or katello 3.18) as is